pkg/k8shandler/common.go

```diff
@@ -26,11 +27,8 @@ func addOwnerRefToObject(o metav1.Object, r metav1.OwnerReference) {
 }
 
 func getImage(commonImage string) string {
-	image := commonImage
-	if image == "" {
-		image = utils.LookupEnvWithDefault("ELASTICSEARCH_IMAGE", constants.ElasticsearchDefaultImage)
-	}
-	return image
+	logger.Infof("Ignoring the image specified by the custom resource. Only this operator is able to coordinate the configuration and the image: %s", commonImage)
```

Comment:
this is going to be very noisy... do we need to print this out?

Reply:
updated with status condition per our conversation